tests/sqlparser_common.rs

```diff
@@ -269,11 +274,20 @@ fn parse_where_delete_statement() {
     match verified_stmt(sql) {
         Statement::Delete {
             table_name,
+            using,
             selection,
-            ..
         } => {
-            assert_eq!(ObjectName(vec![Ident::new("foo")]), table_name);
+            assert_eq!(
+                TableFactor::Table {
+                    name: ObjectName(vec![Ident::new("foo")]),
+                    alias: None,
+                    args: None,
+                    with_hints: vec![]
+                },
+                table_name,
+            );
 
+            assert_eq!(None, using);
```

Comment:
Can you please add a positive test for `using` as well (aka a query that has a `USING` clause)?

Reply:
No this is fine -- I just wanted to add some coverage so that we didn't accidentally break the functionality in the future but have all the tests pass